community.json

```diff
@@ -60,6 +60,12 @@
         "state": "inprogress",
         "url": "https://github.com/YunoHost-Apps/pelican_ynh"
     },
+    "Pixelfed": {
```

Comment:
From what I read [here](https://github.com/YunoHost/apps#contributing), you must use the app id here (hence no uppercase)...